legacy_scripts_backup/docker-manager.sh

```
@@ -0,0 +1,729 @@
+                    echo "Stop container: docker stop $container_name"
+                    echo "Access: http://localhost:$SERVER_PORT"
+                else
+                    echo "Failed to start existing container, creating new one..."
+                    run_container "prod"
+                fi
+            else
+                echo "No existing container found, creating new one..."
+                run_container "prod"
+            fi
+            ;;
+        "cleanup")
```

Shellcheck (0.10.0)

[warning] This pattern never matches because of a previous pattern on line 639.

(SC2222)